manageiq-operator/pkg/helpers/miq-components/httpd_conf.go

```diff
@@ -20,6 +20,8 @@ func httpdApplicationConf(applicationDomain string) string {
 Listen 8080
 # Timeout: The number of seconds before receives and sends time out.
 Timeout 120
+ServerSignature Off
+ServerTokens Prod
```

Comment:
consistency wise, here you put it after the timeout, but above you put it after the listen.  We should be consistent with whatever we did in the manageiq-appliance-build